yellowbrick/features/importances.py

```diff
@@ -207,9 +230,24 @@ def finalize(self, **kwargs):
         # Remove the ygrid
         self.ax.grid(False, axis='y')
 
+        if self.stack:
+            plt.legend(bbox_to_anchor=(1.04, 0.5), loc="center left")
         # Ensure we have a tight fit
         plt.tight_layout()
 
+    def _find_classes_param(self, y):
+        """
+        Searches the wrapped model for the classes_ parameter.
+        """
+        for attr in ["classes_"]:
+            try:
+                return getattr(self.estimator, attr)
+            except AttributeError:
+                continue
+
+        self.stack = False
+        return np.unique(y)
```

Comment:
This was a good way to solve the coloring for each of the stacked pieces, nice problem solve! 

Depending on how you decide to address my earlier comment about using `is_classifier`; it would probably be better for this method to return `None` instead of `np.unique` in the case of a regressor; otherwise, we may store arbitrarily large data; e.g. up to the size of `y` for no purpose.

Reply:
Good point!